requirements_docs.txt

```diff
@@ -0,0 +1,10 @@
+numpy >= 1.12
+scipy >= 1.0, <1.2
+sympy >= 1.2
+toposort
+funcsigs; python_version < '3.0'
+functools32; python_version < '3.0'
```

Comment:
Can't you say that this file should include the normal requirements.txt?

Reply:
I guess we can: https://stackoverflow.com/questions/17803829/how-to-customize-a-requirements-txt-for-multiple-environments